edifice/app.py

```diff
@@ -253,7 +254,7 @@ def _make_widget_helper(comp):
         return _make_widget_helper(self._root)
 
     def start(self):
-        """Start the QEventLoop.
+        """Start the application event loop.
```

Comment:
I think this should be implemented in terms of `start_loop`.

Reply:
Done.